### PR TITLE
[bitnami/containers] Automate license checks

### PR DESCRIPTION
**Pull request by rloporp**
### Description of the change

Licenses and copyright notices are included as headers in all our public repositories.
We'll automatically check license headers with the [SkyWalking Eyes](https://github.com/apache/skywalking-eyes) GitHub action.

### Benefits

Automatic license compliance.

### Possible drawbacks

No known drawbacks.

### Applicable issues

None.

### CODE REVIEW

File: .github/workflows/license-headers.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+name: license-headers
```
Comment (carrodher) [inline .github/workflows/license-headers.yml:5]: We usually add a prefix to order the actions (i.e `[CI/CD] ...`, `[Support] ...`, etc) and a descriptive title

What do you think about using something like this?
```suggestion
name: [License] Check license headers
```

**Review by carrodher — DISMISSED** [review]
LGTM

**Review by gongomgra — CHANGES_REQUESTED** [review]

File: .licenserc.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: VMware, Inc.
+
+    content: |
+      # Copyright VMware, Inc.
+      # SPDX-License-Identifier: APACHE-2.0
+
+  paths:
+    - "**/Dockerfile"
```
[the remark this answers is not in the document]
Comment (rloporp) [reply .licenserc.yaml:16]: I agree we should double-check several other files, but we're limiting the scope to Dockerfiles in this first iteration.

[the remark this answers is not in the document]
Comment (gongomgra) [reply .licenserc.yaml:16]: Perfect then, thanks for the clarification!

**Review by gongomgra — DISMISSED** [review]

**Review by carrodher — APPROVED** [review]

**Pull request merged**